paymaster/src/rpcMethods.ts

```diff
@@ -0,0 +1,54 @@
+import { BigNumberish, ethers } from 'ethers';
+import { paymasterAbi } from './abis';
+import { BytesLike } from '@ethersproject/bytes';
+import dotenv from 'dotenv';
+dotenv.config();
+
+export interface UserOperation {
+    sender: string
+    nonce: BigNumberish
+    initCode: BytesLike
+    callData: BytesLike
+    callGasLimit: BigNumberish
+    verificationGasLimit: BigNumberish
+    preVerificationGas: BigNumberish
+    maxFeePerGas: BigNumberish
+    maxPriorityFeePerGas: BigNumberish
+    paymasterAndData: BytesLike
+    signature: BytesLike
+}
+
+export async function sponsorTransaction(userOp: UserOperation): Promise<UserOperation> {
+    const paymasterAddress = process.env.PAYMASTER_ADDRESS as string;
+    const provider = new ethers.JsonRpcProvider(process.env.RPC_URL as string);
+    const signer = new ethers.Wallet(process.env.PRIVATE_KEY as string);
+
+    const paymaster = new ethers.Contract(
+        paymasterAddress,
+        paymasterAbi,
+        provider
+    );
+
+    /// NOTE: Define additional transaction sponsor logic here if needed
+    /// For example, it could be a list of whitelisted addresses
```

Comment:
Do you think you could implement this as a configuration value that is either `None`/`undefined` (in which case there is not check) or a list (in which case only whitelisted contracts are allowed, nothing allowed if list is empty)?

Probably good for next PR.

Reply:
^ for this one though, `.env` won't cut it :)

Reply:
Yeah, I think this would be a good enhancement. Added a new issue: https://github.com/0xFableOrg/simple-op-stack-rollup/issues/39